### PR TITLE
'Gnuplot'

### PR DESCRIPTION
**Pull request by oplambeck**
Pull Request for chapel-gnuplot to be added to mason-registry

### CODE REVIEW

File: Bricks/Gnuplot/0.0.1.toml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+
+[brick]
+name = "Gnuplot"
+version = "0.0.1"
```
Comment (ben-albrecht) [inline Bricks/Gnuplot/0.0.1.toml:5]: How about an `authors` field with Marcos Cleison and yourself (as the current package maintainer)?

Comment (ben-albrecht) [inline Bricks/Gnuplot/0.0.1.toml:5]: Was this version number inherited from the original source? If not, I'd suggest starting with `0.1.0` as initial version number.

File: Bricks/Gnuplot/0.0.1.toml [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+chplVersion = "1.18.0"
```
Comment (ben-albrecht) [inline Bricks/Gnuplot/0.0.1.toml:6]: Can you confirm the tests pass with Chapel 1.19?

File: Bricks/Gnuplot/0.0.1.toml (hunk 1 above)
Comment (oplambeck) [reply Bricks/Gnuplot/0.0.1.toml:5]: This was inherited from the source, I used 'mason publish' to do this and it just grabs it from the source.

**Review by ben-albrecht — APPROVED** [review]

**Pull request merged**